<?xml version="1.0" encoding="UTF-8"?>
<!-- Vị trí: /ui/widgets/channelmanagerwidget.ui -->
<!-- Phiên bản: 2.2 (Sửa kích thước toolbar) -->
<ui version="4.0">
 <class>ChannelManagerWidget</class>
 <widget class="QWidget" name="ChannelManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ChannelToolbarWidget" name="toolbarWidget" native="true"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="ChannelListWidget" name="listWidget" native="true"/>
     <widget class="QWidget" name="statisticsPlaceholder" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChannelToolbarWidget</class>
   <extends>QWidget</extends>
   <header>widgets/channeltoolbarwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChannelListWidget</class>
   <extends>QWidget</extends>
   <header>widgets/channellistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>

